src/internal/pfsload/file.go

```diff
@@ -1,16 +1,26 @@
-package load
+package pfsload
 
 import (
 	"bytes"
 	"fmt"
 	"io"
 	"math/rand"
 	"path"
-
-	"github.com/pachyderm/pachyderm/v2/src/internal/storage/chunk"
-	"github.com/pachyderm/pachyderm/v2/src/internal/uuid"
 )
 
+const pathSize = 32
+
+var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+// randSeq generates a random sequence of data (n is number of bytes)
+func randSeq(n int, random *rand.Rand) string {
```

Comment:
I think we do something like this in the chunk layer too? Can we put these in `randutil`.  I think right now just UniqueString lives in there.